pkg/scheduler/plugins/rescheduling/rescheduling.go

```diff
@@ -153,8 +153,6 @@ func (rc *Configs) parseArguments(arguments framework.Arguments) {
 	if err != nil {
 		klog.V(4).Infof("Parse rescheduling interval failed. Reset the interval to 5m by default.")
 		rc.interval = DefaultInterval
-	} else {
-		Interval = intervalStr
```

Comment:
I understand that after deleting here, the interval configuration should not take effect.

Reply:
@wangyang0616 IMHO, what real take effects is **rc.interval**. We don't need **Interval** variable now

Reply:
https://github.com/volcano-sh/volcano/blob/a592a009a4c6885cb3720ee5a0bf565db2f9abd8/pkg/scheduler/plugins/rescheduling/rescheduling.go#L95-L98
Here, interval is used to control the real running time of rescheduling.

Reply:
Yes, you are right.